ocs_ci/utility/kms.py

```diff
@@ -578,7 +594,7 @@ def vault_create_policy(self, policy_name=None):
             raise VaultOperationError(
                 f"Failed to create policy f{self.vault_policy_name}"
             )
-        self.vault_path_token = self.generate_vault_token()
+        self.vault_path_token = self.get_vault_token(self)
```

Comment:
You don't need to explicitly pass this to the function anymore.

```suggestion
        self.vault_path_token = self.get_vault_token()
```